airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -86,25 +86,47 @@ def data_field(self) -> str:
 
 # Basic incremental stream
 class IncrementalShopifyStream(ShopifyStream, ABC):
-
     # Getting page size as 'limit' from parrent class
     @property
     def limit(self):
         return super().limit
 
+    # Setting the check point interval to the limit of the records output
     state_checkpoint_interval = limit
-
-    cursor_field = "id"
+    # Setting the default cursor field for all streams
+    cursor_field = "updated_at"
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        return {self.cursor_field: max(latest_record.get(self.cursor_field, 0), current_stream_state.get(self.cursor_field, 0))}
-
-    def request_params(self, stream_state=None, **kwargs):
-        stream_state = stream_state or {}
-        params = super().request_params(stream_state=stream_state, **kwargs)
-        params["since_id"] = stream_state.get(self.cursor_field)
+        return {self.cursor_field: max(latest_record.get(self.cursor_field, ""), current_stream_state.get(self.cursor_field, ""))}
```

Comment:
There are some streams that requires `int` comparison for the `latest_record` and `current_stream_state` such as `collects, order_risks` other streams requires `str` comparison for `updated_at` - therefore we have default inside of InrementalStream comparison as default one, and override - inside of `collects, order_risks`.

if we don't specify the types or specify `None`: we would have issues during the `Full_refresh`
`TypeError: '>' not supported between instances of 'NoneType' and 'str'` - for updated_at
`TypeError: '>' not supported between instances of 'NoneType' and 'int'` - for `collects, order_risks`

Because of the use of `max()` function comparison, when we `full_refresh` we have the current stream state as None or something we specify explicitly.